src/Model/Page.php

```diff
@@ -238,7 +238,7 @@ public function getRouteName()
      */
     public function setPageAlias($pageAlias)
     {
-        if ('_page_alias_' != substr($pageAlias, 0, 12)) {
+        if (null !== $pageAlias && '_page_alias_' != substr((string) $pageAlias, 0, 12)) {
```

Comment:
why this null comparison?

Reply:
hm, you are right, not needed https://3v4l.org/fAJVX because of `(string)`

Reply:
Yeah can't see how that would be useful